packages/modelviewer.dev/examples/scenegraph/index.html

```diff
@@ -766,7 +766,7 @@ <h2 class="demo-title">Materials API</h2>
     }
   };
 
-  modelViewer.addEventListener("click", changeColor);
+  document.addEventListener("click", changeColor);
```

Comment:
Have you tested this whole page? I'm guessing since there are multiple examples, this might have odd behavior. Maybe add a check in `changeColor` to ensure the event came through the right element?

Reply:
> add a check in `changeColor` to ensure the event came through the right element?

You are right. i think my new commit fixes this